packages/legend-extension-external-language-morphir/src/ELMorphir_GenerationPlugin.tsx

```diff
@@ -0,0 +1,173 @@
+/**
```

Comment:
Let's rename this file and class to `ELMorphir_StudioPlugin` and move it to `src/components`

Reply:
Fixed.